lib/game_of_life_web/live/universe_live.ex

```diff
@@ -0,0 +1,62 @@
+defmodule GameOfLifeWeb.UniverseLive do
+  use Phoenix.LiveView
+  use Phoenix.HTML, only: raw/1
+  alias GameOfLife.Universe
+
+  def render(assigns) do
+    ~L"""
+    <div id="universe-page">
+      <h1>It's time for... the Game... of... LIFE!</h1>
+
+      <div class="universe">
+        <%= for row <- @cells do %>
+          <div class="cell-row">
+            <%= for cell <- row do %>
+              <%= raw cell %>
+            <% end %>
+          </div>
+        <% end %>
+      </div>
+    </div>
+    """
+  end
```

Comment:
Okay, I'm just going to say that this ~L `leex` stuff is insane....

... -ly awesome

Reply:
it is pretty cool... I actually prefer having a separate `leex` file.  Then there is no reason for `render_cells`.  The `leex` file can handle the loop